lib/inquisition/fasterer/runner.rb

```diff
@@ -14,6 +15,12 @@ def call
       def scan_file(path)
         analyzer = ::Fasterer::Analyzer.new(path)
         analyzer.scan
+        check_analyser_config_rules(analyzer)
+      end
+
+      def check_analyser_config_rules(analyzer)
+        return unless @fasterer.offenses_grouped_by_type(analyzer).any?
```

Comment:
Why do we need this check?

Reply:
it is check type of analyzer.errors and show, contain this type error in config file or not. if not contain in config this type, this mehtod returne current error, if contain, I mush not create issue with this type error